### ISSUE TITLE
MPPT calculate the efficiency sometimes wrong

### ISSUE THREAD

**Issue by SW-Niko** [helgeerbe/OpenDTU-OnBattery#858]
### What happened?

On MPPT with active DC load output the efficiency is wrong.

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129541740/bada3878-02ee-46c4-8a85-b748e2e072c8)

### To Reproduce Bug

Use the DC load output at night

### Expected Behavior

On MPPT with active DC load the calculation must be done including the load current.

### Install Method

Self-Compiled

### What git-hash/version of OpenDTU?

g0ed09ae

### Relevant log/trace output

```Shell
Screen shot at night.
```

### Anything else?

Minor bug. I will fix the issue in the next days and start PR.

**Comment by schlimmchen** [comment]
The calculated power value also seems wrong, doesn't it?

Can you elaborate what the load output is, what it does, and when it is active? How does current flow when the load output is active?

**Comment by SW-Niko** [comment]
Hello @schlimmchen ,
PR #867 is ready.

Background:
Some MPPT do have an load output. (VE.Direct value IL)
Now you can have different situations:
- All power from the panel flows into the battery
- One part from the panel power flows into the battery one part into the load
- All power from the panel flows into the load
- With no power from the panel all power into the load is taken from the battery.

In short ... to calculate the effectiveness of the MPPT you have to take the battery current and the load current into account.

> The calculated power value also seems wrong, doesn't it?

Yes, In the meantime I learned "Do not calculate with a mix of signed and unsigned integer" 😞
